### DISCUSSION TITLE
Post processing of Hits, api side

### DISCUSSION THREAD

**Discussion by RomainSF** [appbaseio/reactivesearch#1660] [Q&A] [answered]
Hi there,

trying to post-process the data coming from elastic search.
elastic search doesn't have all the data I want, I need to add extra data to each Hits

is there a simple way?

**Comment by bietkul** [comment, marked as answer]
@RomainSF You can try using `transformResponse` prop.
https://docs.appbase.io/docs/reactivesearch/v3/overview/reactivebase/#props
![alt img](https://i.imgur.com/6WmLtT8.png)
